src/main/java/org/nuxeo/audit/storage/impl/SQLAuditStorage.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * (C) Copyright 2017 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Estelle Giuly <egiuly@nuxeo.com>
+ */
+
+package org.nuxeo.audit.storage.impl;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.logging.Log;
```

Comment:
Organize imports should put a blank line there. Please setup your IDE ([eclipse](https://doc.nuxeo.com/corg/eclipse-setup/)/[IDEA](https://doc.nuxeo.com/corg/intellij-idea-setup/))